### PR TITLE
This small plugin adds affordances for print-statement debugging.

### PR DESCRIPTION
**Pull request by obra**

### CODE REVIEW

File: plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//namespace kaleidoscope {
+//namespace plugin {
+//
+//EventHandlerResult ArduinoTrace::onSetup() {
+//  return EventHandlerResult::OK;
+//}
+//}
+//}
+//
+//kaleidoscope::plugin::ArduinoTrace ArduinoTrace;
```
Comment (algernon) [inline plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.cpp:28]: If the plugin doesn't provide anything else other than pulling ArduinoTrace in, do we even need thtis file?

File: plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.h [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+/*
+#pragma once
+
+#include "kaleidoscope/Runtime.h"
+
+#ifndef ARDUINOTRACE_SERIAL
+#define ARDUINOTRACE_SERIAL DebugStderr
+#endif
+
+#include "ArduinoTrace.h"
```
Comment (algernon) [inline plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.h:26]: Is it intended that the entire contents are commented out? If so, I'm not sure I understand how this plugin works...

**Review by algernon — CHANGES_REQUESTED** [review]
The plugin header and cpp both seem to be completely commented out - I'm not sure that's intended.

**Comment by obra** [comment]
Good catch. Those files should have been git rm-ed. I've now force-pushed

**Pull request merged**